components/brave_ads/browser/ads_service_impl.h

```diff
@@ -88,8 +88,6 @@ class AdsServiceImpl : public AdsService,
   void SetAdsPerHour(
       const uint64_t ads_per_hour) override;
 
-  uint64_t GetAdsPerDay() const override;
```

Comment:
I did a quick search in b-c, is it correct that `GetAdsPerDay` was never used? How did we enforce  ads per day before if `GetAdsPerDay` was never called?

Reply:
This was pref was used in the frequency cap. However, there was no need to expose this value via a pref as the value was never exposed in the UI. In the Muon days you could set ads per hour and ads per day in the UI.